httpbin/core.py

```diff
@@ -293,7 +293,7 @@ def view_headers():
         description: The Rrquest's IP Address.
     """
 
-    return jsonify(get_dict('headers'))
+    return jsonify(get_headers())
```

Comment:
@sigmavirus24, it looks like `get_dict('headers')` is already using `get_headers` [under the hood](https://github.com/requests/httpbin/blob/63a649f753e625feac9196115a8ba74fd36864fe/httpbin/helpers.py#L192). It also already supports a `show_env` arg which seems to work on production httpbin right now. Are we sure this change is needed?